Assets/Scripts/UI/CoreGameplay/Stage/StagePosition.cs

```diff
@@ -9,6 +9,9 @@ public class StagePosition : MonoBehaviour
     public static event Action<StagePosition> OnStagePositionClicked;
     public static event Action<StagePosition> OnStagePositionChanged;
 
+    [Header("Stage Parameters")]
+    public int stagePositionNumber;
```

Comment:
I thought we could use this for a variety of reasons but primarily used for builtClips indexing

Reply:
Another good positive for this - great for analytics (working with @JamesCoburn246  on that now)